content/en/docs/tasks/traffic-management/locality-load-balancing/before-you-begin/index.md

```diff
@@ -157,3 +157,31 @@ Only one of the load balancing options should be configured, as they are
 mutually exclusive. Attempting to configure both may lead to unexpected
 behavior.
 {{< /warning >}}
+
+{{< tip >}}
+In a single multi-zone cluster environment, locality load balancing can also be configured for failover to a different zone within the same cluster.
+To test it, you will need to create a cluster with multiple worker zones and deploy an istiod instance and the app to each zone.
+
+1) If you don’t have a multi-zone Kubernetes cluster, you can deploy one locally using `kind` with the following command:
+
+{{< text bash >}}
```

Comment:
Ideally we would test these snips, but since the test framework doesn't really support customizing the test cluster like this and since it's just a tip, you should probably suppress generation of the snips.
```suggestion
{{< text syntax=bash snip_id=none >}}
```

Reply:
updated